hdbt.theme

```diff
@@ -169,6 +134,32 @@ function hdbt_preprocess_html(&$variables): void {
 
   // Set theme path variable.
   $variables['theme_path'] = _hdbt_get_theme_path();
+
+  // Add helfi-environment as a JS variable.
+  if (\Drupal::hasService('helfi_api_base.environment_resolver')) {
+    try {
+      /** @var \Drupal\helfi_api_base\Environment\EnvironmentResolverInterface $service */
+      $service = \Drupal::service('helfi_api_base.environment_resolver');
+
+      $variables['#attached']['drupalSettings']['helfi_instance_name'] = $service->getActiveProject()
+        ->getName();
+    }
+    catch (\InvalidArgumentException) {
+    }
+  }
+
+  // Toggle between global and local navigation in twig templates.
+  $variables['use_global_navigation'] = \Drupal::moduleHandler()
```

Comment:
This can be optimized further with 
```php
$navigation_enabled = \Drupal::moduleHandler()->moduleExists('helfi_navigation');
$variables['use_global_navigation'] = $navigation_enabled;
$variables['#attached']['drupalSettings']['hdbt']['global_menu'] = $navigation_enabled;
$variables['#attached']['drupalSettings']['hdbt']['search_dropdown'] = $navigation_enabled;
```